pkg/deploy/api/types.go

```diff
@@ -405,6 +405,10 @@ type DeploymentConfigList struct {
 // DeploymentConfigRollback provides the input to rollback generation.
 type DeploymentConfigRollback struct {
 	unversioned.TypeMeta
+	// Name of the deployment config that will be rolled back.
+	Name string
```

Comment:
Even though we don't really need to follow what upstream is doing here
since we will have a different path for rolling back, I am using the same
fields
https://github.com/kubernetes/kubernetes/blob/1d128614bc463752ce0233fcee11d8bb9778963e/pkg/apis/extensions/v1beta1/types.go#L263-L266
Should subresources have object metadata? They are not supposed to be first
class objects, right?

On Tue, Jun 7, 2016 at 5:35 PM, Clayton Coleman <notifications@github.com>
wrote:

> In pkg/deploy/api/types.go
> <https://github.com/openshift/origin/pull/9135#discussion_r66095217>:
>
> > @@ -405,6 +405,10 @@ type DeploymentConfigList struct {
> >  // DeploymentConfigRollback provides the input to rollback generation.
> >  type DeploymentConfigRollback struct {
> >  	unversioned.TypeMeta
> > +	// Name of the deployment config that will be rolled back.
> > +	Name string
>
> You should be using ObjectMeta if you're going to do this.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9135/files/0a31c175b6640862d4e1303a260629570af5f236#r66095217>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADuFf5_xsrUW2XoAftOPijVHUdCTgbyCks5qJY_cgaJpZM4IsjxM>
> .
>